src/components/TDViewer/TDViewer.jsx

```diff
@@ -64,20 +74,39 @@ export default function TDViewer() {
         }
     }
 
+    const onClickAddGlobalForm = async () => {
+        const formToAdd = await addGlobalForm();
+        if (formToAdd) {
+            if (!tdJSON['forms']) {
+                tdJSON.forms = [];
+            }
+            tdJSON.forms.push(formToAdd)
+            context.updateOfflineTD(JSON.stringify(tdJSON, null, 2))
```

Comment:
I think it's worth extracting and reusing this magic constant

Reply:
Constant is extracted